contracts/src/starknet/rewaave/math/wad_ray_math.cairo

```diff
@@ -0,0 +1,218 @@
+from starkware.cairo.common.uint256 import (
```

Comment:
from where does this code come from? should we test it?

Reply:
This is a direct tranpsilation of the implementation in Solidity. There's already a ticket about testing it in the board (added it after the meeting). This is the original source: https://github.com/aave/protocol-v2/tree/1d80f9a860b5d389fbae9034338c01fac6f15aa2/contracts/protocol/libraries/math